x/cash/dynamicfee.go

```diff
@@ -144,39 +109,61 @@ func (d DynamicFeeDecorator) chargeFee(store weave.KVStore, src weave.Address, a
 	return d.ctrl.MoveCoins(store, src, dest, amount)
 }
 
-// prepare is all shared setup between Check and Deliver, one more level above extractFee
-func (d DynamicFeeDecorator) prepare(ctx weave.Context, store weave.KVStore, tx weave.Tx) (fee x.Coin, payer weave.Address, cache weave.KVCacheWrap, err error) {
-	var finfo *FeeInfo
-	fee = x.Coin{}
+// chargeMinimalFee deduct an anty span fee from a given account.
+func (d DynamicFeeDecorator) chargeMinimalFee(store weave.KVStore, src weave.Address) error {
```

Comment:
@ethanfrey  This is something that was not included in the original pull request. If we support zero fees without a ticker then an introspection of the account of the payer is required.

Reply:
Adressed in https://github.com/iov-one/weave/pull/364/commits/5bdc3babe4b936726d84774564b7758710125673